src/concrete/ml/onnx/convert.py

```diff
@@ -208,7 +221,13 @@ def get_equivalent_numpy_forward_from_onnx(
             f"Available ONNX operators: {', '.join(sorted(IMPLEMENTED_ONNX_OPS))}"
         )
 
+    # Compute the LSB to be remove manually while waiting for the truncate feature
```

Comment:
maybe say it the other way to be more explicit , like : `Remove this workaround (which computes the LSB to be remove manually) once the truncate feature is released`

Reply:
The workaround is to compute the LSB manually  (using the bitwidth precision from the onnx graph )instead of using the autorounder.

The truncate and auto truncate will look like rounding_bit_pattern and autorounder according to Umut.

Reply:
> Maybe say it the other way to be more explicit , like : Remove this workaround (which computes the LSB to be remove manually) once the truncate feature is released

OK